src/Makefile.am

```diff
@@ -2757,3 +2757,18 @@ tests_testEventLoop_LDADD = \
 	$(COMPAT_LIB) \
 	$(XTRA_LIBS)
 tests_testEventLoop_LDFLAGS = $(LIBADD_DL)
+
+check_PROGRAMS += tests/testIoManip
```

Comment:
```
FAIL: tests/testIoManip
```

@kinkie, if you do not mind, please check why https://github.com/squid-cache/squid/actions/runs/7021325784?pr=1578 CI logs do not show any details about the current testIoManip failure. If I just failed to find those details in the logs, please let me know what I missed. I wonder if there is something wrong with our CI scripts that hides those details...

I do not insist on this investigation or any related changes (because I do not know whether the problem, if any, is related to this PR).

Reply:
dismissing stale comment; tests are run and passing

Reply:
And by the way: thanks for improving the tests to the level you did, @rousskov !

Reply:
> dismissing stale comment; tests are run and passing

Please note that my request was _not_ about fixing not running or failing tests. It was about our CI logs missing test failure details. Sooner or later, testIoManip will fail for some PR. It would be nice to avoid dealing with the same problem then. However, it is possible that the "missing failure details" problem, whatever it was, has been unknowingly fixed since then.

I do not insist on any changes related to this change request.

Reply:
I tried injecting a failure into testIoManip, here is the outcome of the test:

```
[...]
PASS: tests/testEventLoop
FAIL: tests/testIoManip
============================================================================
Testsuite summary for Squid Web Proxy 7.0.0-VCS
============================================================================
# TOTAL: 36
# PASS:  35
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See src/test-suite.log
Please report to https://bugs.squid-cache.org/
============================================================================
make[4]: *** [Makefile:6202: test-suite.log] Error 1
make[4]: Leaving directory '/home/kinkie/src/squid/btbuild/src'
make[3]: *** [Makefile:6310: check-TESTS] Error 2
make[3]: Leaving directory '/home/kinkie/src/squid/btbuild/src'
make[2]: *** [Makefile:6653: check-am] Error 2
make[2]: Leaving directory '/home/kinkie/src/squid/btbuild/src'
make[1]: *** [Makefile:6094: check-recursive] Error 1
make[1]: Leaving directory '/home/kinkie/src/squid/btbuild/src'
make: *** [Makefile:6655: check] Error 2
make: Leaving directory '/home/kinkie/src/squid/btbuild/src'
```

The test framework left behind ```src/tests/testIoManip.log ``` and ```src/tests/testIoManip.trs```

src/tests/testIoManip.log contains:
```
SKIP: mem/libmem.la MemPools (not implemented).
.F
testIoManip.cc:155:Assertion
Test name: TestIoManip::testAsHex
equality assertion failed
- Expected: 1
- Actual  : 0

Failures !!!
Run: 1   Failure total: 1   Failures: 1   Errors: 0
FAIL tests/testIoManip (exit status: 1)
```

and src/tests/testIoManip.trs contains
```
:test-result: FAIL
:global-test-result: FAIL
:recheck: yes
:copy-in-global-log: yes
```

It should be easy to add a reporting step for failures